src/plots/plots.js

```diff
@@ -1549,8 +1549,14 @@ plots.extendObjectWithContainers = function(dest, src, containerPaths) {
         for(i = 0; i < containerPaths.length; i++) {
             containerProp = Lib.nestedProperty(expandedObj, containerPaths[i]);
             containerVal = containerProp.get();
-            containerProp.set(null);
-            Lib.nestedProperty(containerObj, containerPaths[i]).set(containerVal);
+
+            if(!containerVal) {
```

Comment:
> This is fine because this could only reasonably be an object or not an object, right?

Correct. That's what I'm thinking.

Maybe we could be slightly more strict and make this condition `if(containerVal === undefined)` which would be fulfilled when `{ annotations: undefined }` and `{ annotations: null }` but not for other falsy values.